pkgs/applications/audio/parlatype/default.nix

```diff
@@ -0,0 +1,62 @@
+{ lib, stdenv, fetchFromGitHub, pkg-config, meson, gtk3, dbus, gst_all_1, ninja, gettext, appstream-glib, python3, desktop-file-utils, glib, gobject-introspection, gsettings-desktop-schemas, isocodes, itstool, libxml2, wrapGAppsHook, hicolor-icon-theme }:
+
+stdenv.mkDerivation rec {
+  pname = "parlatype";
+  version = "3.1";
+
+  src = fetchFromGitHub {
+    owner  = "gkarsay";
+    repo   = pname;
+    rev    = "v${version}";
+    sha256 = "1a4xlsbszb50vnz1g7kf7hl7aywp7s7xaravkcx13csn0a7l3x45";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    meson
+    ninja
+    gettext
+    appstream-glib
+    python3
+    desktop-file-utils
+    gobject-introspection
+    itstool
+    libxml2
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    gtk3
+    dbus
+    gst_all_1.gstreamer
+    gst_all_1.gst-plugins-base
+    gst_all_1.gst-plugins-good
+    gst_all_1.gst-plugins-bad
+    gst_all_1.gst-plugins-ugly
+    gst_all_1.gst-libav
+    glib
+    gsettings-desktop-schemas
+    hicolor-icon-theme
+    isocodes
+  ];
+
+  postPatch = ''
+    patchShebangs data/meson_post_install.py
+    patchShebangs libparlatype/tests/data/generate_config_data
+  '';
+
+  doCheck = false;
```

Comment:
Why?

Reply:
Because it was like that in the original package :) It seems that the tests require X. Is there a way to run that?

(At least one of them seems to want to interact with the system volume (?) and fails on my machine about half the time, I don't know what that is about.)